packages/gestalt/src/Button.jsdom.test.js

```diff
@@ -8,28 +8,17 @@ describe('Button', () => {
     const mockOnClick = jest.fn<
       [
         {|
-          dangerouslyDisableOnNavigation: () => void,
-          event:
-            | SyntheticMouseEvent<HTMLButtonElement>
-            | SyntheticMouseEvent<HTMLAnchorElement>
-            | SyntheticKeyboardEvent<HTMLAnchorElement>
-            | SyntheticKeyboardEvent<HTMLButtonElement>,
+          event: SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement>,
         |},
       ],
       void,
     >();
-    render(<Button text="ButtonText" onClick={mockOnClick} />);
-    screen.getByText('ButtonText').click();
+    const { getByText } = render(<Button text="ButtonText" onClick={mockOnClick} />);
+    // eslint-disable-next-line testing-library/prefer-screen-queries -- Please fix the next time this file is touched!
+    getByText('ButtonText').click();
     expect(mockOnClick).toHaveBeenCalled();
   });
 
-  it('renders a submit button and forwards a ref to the innermost <button> element', () => {
```

Comment:
let;s not remove this

Reply:
No idea why it got removed heheh
Added it back.